mozjs/src/gc/root.rs

```diff
@@ -153,11 +154,14 @@ impl<'a, T> Handle<'a, T> {
     where
         T: Copy,
     {
-        *self.ptr
+        unsafe { *self.ptr }
     }
 
     pub(crate) fn new(ptr: &'a T) -> Self {
```

Comment:
Leaving this function signature alone is suspicious. If ptr is being made from an `&T` we can't write through it (according to the memory models) and we presumably end up with the same issue at a distance.

Reply:
I think this is ok for immutable handle, that will live for lifetime of `'a`.